lib/src/jira/api.rs

```diff
@@ -53,6 +60,12 @@ pub struct JiraSession {
     fix_versions_field: String,
     pending_versions_field: Option<String>,
     pending_versions_field_id: Option<String>,
+    release_note_text_field: Option<String>,
+    release_note_text_field_id: Option<String>,
+    release_note_channels_field: Option<String>,
+    release_note_channels_field_id: Option<String>,
+    release_note_status_field: Option<String>,
+    release_note_status_field_id: Option<String>,
```

Comment:
```suggestion
    release_note_text_field_id: Option<String>,
    release_note_channels_field_id: Option<String>,
    release_note_status_field_id: Option<String>,
```

The non-id fields are  never read.

```warning: fields `release_note_text_field`, `release_note_channels_field`, and `release_note_status_field` are never read
  --> lib/src/jira/api.rs:63:5
   |
58 | pub struct JiraSession {
   |            ----------- fields in this struct
...
63 |     release_note_text_field: Option<String>,
   |     ^^^^^^^^^^^^^^^^^^^^^^^
64 |     release_note_text_field_id: Option<String>,
65 |     release_note_channels_field: Option<String>,
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
66 |     release_note_channels_field_id: Option<String>,
67 |     release_note_status_field: Option<String>,
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(dead_code)]` on by default
```

Reply:
You're right @rcorre I added some cli capability near the end so I could test end to end capability against github and Jira and it looks like I did not get all of that cleaned up, I'll go over this again to clean up

Reply:
Okay I removed the code related to fetching the jira fields, see latest commit.